java-checks-test-sources/default/src/main/java/symbolicexecution/checks/OptionalGetBeforeIsPresentCheck_jdk11.java

```diff
@@ -49,4 +51,105 @@ private void usingIsEmpty5() {
     }
     op.get(); // Noncompliant {{Call "op.isPresent()" or "!op.isEmpty()" before accessing the value.}}
   }
+
```

Comment:
There are some missing samples for the signatures:
`org.junit.jupiter.api.Assertions#assertTrue(Z)V`,
`org.junit.jupiter.api.Assertions#assertFalse(ZLjava/lang/String;)V`
`org.junit.Assert#assertTrue(Z)V`
`org.junit.Assert#assertFalse(Ljava/lang/String;Z)V`

```
  void test_junit5_0(Optional<String> op){
    Assertions.assertTrue(op.isPresent());
    op.get();  // Compliant
  }
  
  void test_junit5_0_1(Optional<String> op){
    Assertions.assertFalse(op.isPresent(), "Hello");
    op.get();  // Noncompliant
  }
  
  void test_junit4_1_2(Optional<String> op){
    Assert.assertTrue(op.isPresent());
    op.get();  // Compliant
  }
  
  void test_junit4_4_2(Optional<String> op){
    Assert.assertFalse("", op.isEmpty());
    op.get(); // Compliant
  }
```